stargate-blog/tutorial/04-http-create-post.md

```diff
@@ -0,0 +1,153 @@
+---
+order: 4
+---
+
+# Create posts with HTTP
```

Comment:
We don't need this section anymore. Fetching unsigned tx, signing and broadcasting through `/txs` has been replaced with proto and broadcasting through CosmJS (which uses Tendermint's endpoint directly).